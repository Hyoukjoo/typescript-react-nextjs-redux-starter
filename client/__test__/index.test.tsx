import React from 'react';
import { shallow } from 'enzyme';

import Button from '@components/button';
import BUTTON from '@constants/button';

describe('<Index />', () => {
  describe('<Button />', () => {
    const wrapper = shallow(<Button>{BUTTON.DONE}</Button>);

    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });

    it('should get children', () => {
      expect(wrapper.find('i').text()).toBe('done');
    });
  });
});
