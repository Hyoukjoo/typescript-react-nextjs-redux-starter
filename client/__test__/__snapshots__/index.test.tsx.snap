// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> <IconButton /> should render without throwing an error 1`] = `
<Styled(button)>
  <i
    className="material-icons"
  >
    done
  </i>
</Styled(button)>
`;
